scripts/rmc.py

```diff
@@ -192,10 +192,24 @@ def compile_single_rust_file(
 def cargo_build(crate, target_dir, verbose=False, debug=False, mangler="v0", dry_run=False, symbol_table_passes=[]):
     ensure(os.path.isdir(crate), f"Invalid path to crate: {crate}")
 
-    rustflags = rustc_flags(mangler, symbol_table_passes)
+    def get_config(option):
+        process = subprocess.run(
+            [RMC_RUSTC_EXE, option],
+            universal_newlines=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            env=os.environ,
+            cwd=crate)
```

Comment:
What about wrapping this into a `run_cmd` call? Otherwise `--dry-run` will not print this command (not sure if it works with cargo commands anyway).

Reply:
That's a good question. I was thinking that this is a different use case. This is just to get the correct tool configuration so I'm not sure we want to include that in the --dry-run output.  The other problem is how `run_cmd` handles the process output. I didn't want to add yet another option to `run_cmd` that would change how the output is handled.

Reply:
Right, this is not something we need in `--dry-run`.